widgets-samples/cc/samples-cc-react-app/src/App.tsx

```diff
@@ -201,18 +202,36 @@ function App() {
                   <legend className="legend-box">&nbsp;Select Widgets to Show&nbsp;</legend>
                   <div className="widget-checkboxes">
                       {Object.keys(defaultWidgets).map((widget) => (
-                        <>
-                          <label key={widget}>
-                            <input
-                              type="checkbox"
-                              name={widget}
-                              checked={selectedWidgets[widget]}
-                              onChange={handleCheckboxChange}
-                            />
-                            &nbsp;{widget.charAt(0).toUpperCase() + widget.slice(1).replace(/([A-Z])/g, ' $1')}&nbsp;
-                          </label>
-                        </>
-                      ))}
+                          <>
+                            <label key={widget}>
+                              <input
+                                type="checkbox"
+                                name={widget}
+                                checked={selectedWidgets[widget]}
+                                onChange={handleCheckboxChange}
+                              />
+                              &nbsp;
+                              {widget.charAt(0).toUpperCase() + widget.slice(1).replace(/([A-Z])/g, ' $1')}&nbsp;
+                              {widget === 'outdialCall' && (
+                                <span style={{display: 'inline-flex', alignItems: 'center'}}>
+                                  <PopoverNext
+                                  trigger="mouseenter"
+                                  triggerComponent={<Icon name="info-badge-filled" />}
+                                  placement="auto-end"
+                                  closeButtonPlacement="top-left"
+                                  closeButtonProps={{'aria-label': 'Close'}}
+                                  >
+                                  <Text>
+                                    <div style={{color: 'var(--mds-color-theme-text-error-normal)', marginBottom: '10px'}}>
+                                    <strong>Note:</strong> You need to select Incoming Task, Call Control before selecting Outdial Call
```

Comment:
Is this a prerequisite?

What if Im logged into on extension. and then make an outdial call -> the call will come on the extension and I can pick it up there. Putting this means we are saying outdial is dependent on the call-controll and incoming-task widget, which is not desirable.

Callcontrol also is not required, we can control the call from somewhere else it doesnt have to be at the same place as the outdial widget.

Reply:
Yes. The text should reflect it precisely why we put this here as discussed during our Bi-Weekly Sync
```suggestion
                                    <strong>Note:</strong> When a number is dialed, the agent gets an incoming task to accept via an Extension, Dial Number, or Browser. It's recommended to have the incoming task/task list widget and call controls widget according to your needs.
```

Reply:
@Shreyas281299 Yes, this is a prerequisite which is required for all 3 modes of login, even if it is an ext or DN mode we need to show Incoming Task and Call Control Widgets, So we need to check these widgets before outDial Widgets.

Reply:
Message Update to this:- **When a number is dialed, the agent gets an incoming task to accept via an Extension, Dial Number, or Browser. It's recommended to have the incoming task/task list widget and call controls widget according to your needs.**

Reply:
> @Shreyas281299 Yes, this is a prerequisite which is required for all 3 modes of login, even if it is an ext or DN mode we need to show Incoming Task and Call Control Widgets, So we need to check these widgets before outDial Widgets.

 I slightly disagree. it is not a prerequisite becuase dialer module is only used to make a call now where this call is received or accepted from is left on the ext devs and it not mandatory for Incoming task and call control to be present with outdial module

Reply:
Test the scenario and verified its working as expected
- Made call from 1 tab (only dialer module is rendered)
- Received and accepted the call from 2nd tab